src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/store/RemovingDraftTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.draftstore.features.store;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.draftstore.SmokeTestSuite;
+import uk.gov.hmcts.reform.draftstore.response.Draft;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestPropertySource("classpath:application.properties")
+@RunWith(SpringRunner.class)
+public class RemovingDraftTest extends SmokeTestSuite {
```

Comment:
In one of the future pull requests it would be good to change `SmokeTestSuite` to `EndToEndTestSuite`, `AatTestSuite` or sth similar.